src/wayland_client_core.zig

```diff
@@ -125,3 +125,39 @@ pub const EglWindow = opaque {
     extern fn wl_egl_window_get_attached_size(egl_window: *EglWindow, width: *c_int, height: *c_int) void;
     pub const getAttachedSize = wl_egl_window_get_attached_size;
 };
+
+pub const CursorTheme = opaque {
+    extern fn wl_cursor_theme_load(name: ?[*:0]const u8, size: c_int, shm: *client.wl.Shm) ?*CursorTheme;
+    pub const load = wl_cursor_theme_load;
+
+    extern fn wl_cursor_theme_destroy(wl_cursor_theme: *CursorTheme) void;
+    pub const destroy = wl_cursor_theme_destroy;
+
+    extern fn wl_cursor_theme_get_cursor(theme: *CursorTheme, name: [*:0]const u8) ?*Cursor;
+    pub const getCursor = wl_cursor_theme_get_cursor;
+};
+
+pub const Cursor = extern struct {
+    image_count: c_uint,
+    images: [*]*CursorImage,
+    name: ?[*:0]u8,
```

Comment:
libwayland deferences this without checking in `wl_cursor_theme_get_cursor()`
```suggestion
    name: [*:0]u8,
```